services/checks/checks.go

```diff
@@ -115,6 +118,20 @@ func New(agentsRegistry agentsRegistry, alertsRegistry alertRegistry, db *reform
 		publicKeys: defaultPublicKeys,
 		interval:   defaultInterval,
 		startDelay: defaultStartDelay,
+
+		mExecutedScripts: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: prometheusNamespace,
+			Subsystem: prometheusSubsystem,
+			Name:      "check_script_executed_total",
+			Help:      "Counter of check scripts executed per service type",
+		}, []string{"service_type"}),
+
+		mGeneratedAlerts: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: prometheusNamespace,
+			Subsystem: prometheusSubsystem,
+			Name:      "generated_alerts_total",
```

Comment:
`alerts_generated_total` to match the order of words of the metric above.

Please also consider renaming field names to match metric names in word order.